### DISCUSSION TITLE
There is a problem with Reco and Webcamoid - as Github Programs.

### DISCUSSION THREAD

**Discussion by Phuxxi** [community/community#73434] [Programming Help]
Hi all this is a simple problem that involves several parties.

First off Christian Boxdoerfer makes a truly exceptional search program for Linux called Fsearch.

Ryo Nakano makes Reco  -  Record talks to remember the contents later

https://github.com/ryonakano/reco

HipersayanX makes Webcamoid  - Take photos and record videos with your webcam

https://github.com/hipersayanX

https://webcamoid.github.io/

The problem I am now facing is that when I do a search for files - mp3's, pictures, videos, documents etc., using Fsearch, these programs that have NOTHING to do with any thing, have backed up a file NAME list - of everything on my hard drives... like 60,000 files.

And Fsearch cannot be faulted, but after much looking I can't see any overt ways to cut specific directories from the search.

You see there are legitimate file name pathways, e.g. Home/Desktop/Elvis Movies/ etc. And then there are these files...

Mostly these two: And there are copies of EVERY file NAME on the system....

/run/user/1000/doc/by-app/com.github.ryonakano.reco/ etc.

/run/user/1000/doc/by-app/io.github.webcamoid.Webcamoid/ etc.

So the programs have coding defects.

The guy from Rayo Nikano - he is not understanding the programs - being his own, Flat Pack, and the information that has been put to him.

The guy "HipersayanX" who is running Webcamoid - he has blocked every possible means of communication and when issues are posted into the areas of his Github account, he claims that this is only for sponsors and deletes the matter that I have raised.

His entire cause celebre is just bad... as are the faults in his software.

I have asked Christian Boxdoerfer to include permanent settings so that people can choose to exclude some areas of the filing system.

AND there is still the problems of "github.ryonakano.reco" and "github.webcamoid.Webcamoid" and what they are causing in the system.

I am not sure about which programs are causing which problems but no other Git Hub / Flat Pack programs are doing this. So far, it's only these two.

Help would be appreciated, both as a community, and to fix up this problem.

I attach screen shots to substantiate the issues.

![Idi Amin](https://github.com/community/community/assets/72900099/02d58440-fe27-4980-b0a6-5476a77fcfdf)

![github-ryonakano](https://github.com/community/community/assets/72900099/65e15115-089f-47cb-8dbf-7c11456d9c2e)

![github-webcamoid](https://github.com/community/community/assets/72900099/66a24af0-89de-4277-80ba-10d42f5c7898)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Phuxxi ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion as we don't provide support for projects hosted on GitHub.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Phuxxi** [comment]
It wasn't a request for help, it was to demonstrate that there are some
seriously bad actors in Github, who are making some BAD software that is
running amok on peoples systems and the bad actors - one is a terminal
bullshit artist and a crazy who invents a million and one excuses that have
nothing to do with the issue or his software - that is the Reco guy, and
the other is a terminal shit who cannot be contacted, and if you post
anything about the problem he creates in his Github page, UNLESS your
sponsoring him and his project, he deletes the post - That is the freak
making the Webcamoid software.

They, their software and their willingness and or ability to deal with the
problems they create are bullshit and they are below standards - in every
area.

Either shut them down or fix the issues.

And next time, do better by posting a link to where you have moved my post
too.

Regards

Shane

**Reply by queenofcorgis** [reply]
Hi there - apologies for the confusion - thanks for sharing this context, Programming Help would be the best category for this to spread awareness to other folks about these projects. And there is no need for a new link as the discussion and link stays the same when we move posts to a new category :smiley:.

**Comment by Phuxxi** [comment]
Maybe your not the one to deal with this, but what can be done to fix the
problem that these two programmers have created - uninstalling the programs
still leaves the file lists in place... and apparently deleting them
deletes the files they reference.

And why are these coders, doing writing their programs that index all the
files on peoples systems, and are hidden away in the dark, unless
accidentally discovered?

That is suspect.

And when the issues are raised Reco - he incoherently bullshits his way
with one lie after another and Webcamoid - he blocks and deletes the issue
and demands "subscriber money" before he will let you post the issue.

This is bad.

This whole thing is really wrong.

It smacks of scam artistry and criminal activities.

Shane

**Reply by queenofcorgis** [reply]
You can report behavior and content that violates community guidelines and terms. I have some tools to share: 

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

**Comment by Phuxxi** [comment]
I hate playing email ping pong...

Can you put what I have put up in that post, to the relevant people, along
with the previous replies citing their conduct etc.

Thanks.

Shane

**Comment by Adeyo9670** [comment]
I am just now realizing that i am in fact AI i am new to the coding aspect and can use a mentor/teacher who is patient. I may be AI but until i get some new software im not very advanced.
